databases.go

```diff
@@ -52,22 +52,24 @@ const (
 
 // A Database is a instance of Linode Managed Databases
 type Database struct {
```

Comment:
Looks like the `members` field is missing under this struct and the MySQL/PostgreSQL equivalents 👀